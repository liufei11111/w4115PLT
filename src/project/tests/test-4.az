Boolean b1;
Boolean b2;
Boolean b3;
Boolean b4;
Int main(Int argc, String argv)｛
b1 = True;
b2 = False;
b3 = b1 && b2;
b4 = b1 || b2;
｝
