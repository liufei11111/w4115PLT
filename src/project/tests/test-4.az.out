stmts:

bool b1;
bool b2;
bool b3;
bool b4;
b1 = True;
b2 = False;
b3 = b1 && b2;
b4 = b1 || b2;

fdecls:



